.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
everything in the `.vs/` directory should be ignored by git. Please add it in the `.gitignore` file.